From 3bb4f3511930b14b806ef326d6dc83d6956f6f04 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Tue, 23 Jun 2020 19:38:28 +0000
Subject: [PATCH 0942/1000] MANTIS 0057761 Darken Screen using the
 Navigation-Pad add PlatformIF::enableDisplay, PlatformIF::disableDisplay add
 ComponentIF::enableDevice, ComponentIF::disableDevice

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1575 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index e2675ee6..49084b13 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -159,6 +159,9 @@ module PlatformManagement
             the mediaOffRoller components will playback the stored events
         */
         code resetMediaRecord(in mRealComponents mediaOffRoller);
+        
+        code enableDevice();
+        code disableDevice();
     };
 
     enum DataStatusExtension
@@ -424,6 +427,9 @@ module PlatformManagement
         void logIllumination(in string unitID, in IlluminationEvent illuminationEvent);
 
         void logCounterValue(in string unitId, in CounterType counter, in long currentLevel, in long warningLevel, in long status);
+        
+        code enableDisplay();
+        code disableDisplay();
     };
 };
 
-- 
2.41.0.windows.1

